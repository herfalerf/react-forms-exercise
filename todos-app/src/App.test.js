import { render, screen } from "@testing-library/react";
import App from "./App";

it("renders learn react link", () => {
  render(<App />);
});

it("matches snapshot", function () {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
