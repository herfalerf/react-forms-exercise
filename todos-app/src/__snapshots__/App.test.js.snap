// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <form>
        <label
          for="task"
        >
          Task:
        </label>
        <input
          id="task"
          name="task"
          type="text"
          value=""
        />
        <button>
          Add a new todo
        </button>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
